### DISCUSSION TITLE
Import suggestions lacks package path / definition

### DISCUSSION THREAD

**Discussion by gluonfield** [zed-industries/zed#6856] [Feature Requests]
When using a type which is not imported to file yet Zed doesn't show the packages/definitions of the type. This makes it difficult to auto import needed components when names collide.

Zed
<img width="751" alt="image" src="https://github.com/zed-industries/zed/assets/5672094/bfa38e5c-e7c0-41c7-92cc-a45ccf222ebd">

VSCode
<img width="765" alt="image" src="https://github.com/zed-industries/zed/assets/5672094/96c6b40e-35f6-4705-99b1-498f6e8ea5b4">

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
